rfcs/2020-08-31-3640-nginx-metrics-source.md

```diff
@@ -0,0 +1,162 @@
+# RFC 3640 - 2020-08-31 - Nginx HTTP metrics source
+
+This RFC is to introduce a new metrics source to consume metrics from the
+[Nginx HTTP Server](https://www.nginx.com/). The high level plan is
+to implement a scrapper similar to the existing [prometheus
```

Comment:
I think you mean "scraper", unless this is intended to fight with the server. :stuck_out_tongue:

Reply:
LOL. :)